daemon/graphdriver/overlay/overlay.go

```diff
@@ -181,23 +176,27 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 		locker:        locker.New(),
 	}
 
-	return NaiveDiffDriverWithApply(d, uidMaps, gidMaps), nil
-}
-
-func parseOptions(options []string) (*overlayOptions, error) {
-	o := &overlayOptions{}
 	for _, option := range options {
-		key, _, err := parsers.ParseKeyValueOpt(option)
+		key, val, err := parsers.ParseKeyValueOpt(option)
 		if err != nil {
 			return nil, err
 		}
 		key = strings.ToLower(key)
 		switch key {
+		case "copy.concurrency":
```

Comment:
This option won't be available to the `overlay2` storage driver, correct?

If so; wondering if we should add it to the `overlay` storage driver, given that we marked that driver deprecated.